client/src/components/Donation/DonateForm.js

```diff
@@ -305,25 +361,62 @@ class DonateForm extends Component {
   }
 
   resetDonation() {
-    return this.setState({ ...initialState });
+    return this.props.updateDonationFormState({ ...defaultDonationFormState });
   }
 
   renderCompletion(props) {
     return <DonateCompletion {...props} />;
   }
 
-  render() {
+  renderModalForm() {
+    const { donationAmount, donationDuration, stripe } = this.state;
     const {
-      donationState: { processing, success, error }
-    } = this.state;
-    if (processing || success || error) {
-      return this.renderCompletion({
-        processing,
-        success,
-        error,
-        reset: this.resetDonation
-      });
-    }
+      handleProcessing,
+      defaultTheme,
+      addDonation,
+      postChargeStripe
+    } = this.props;
+    const donationPlan = `$${donationAmount / 100} / ${donationDuration}`;
+
+    return (
+      <Row>
+        <Col lg={8} lgOffset={2} sm={10} smOffset={1} xs={12}>
+          <Spacer />
+          <b>Confirm your donation of {donationPlan} with PayPal:</b>
+          <Spacer />
+          <PaypalButton
+            addDonation={addDonation}
+            donationAmount={donationAmount}
+            donationDuration={donationDuration}
+            handleProcessing={handleProcessing}
+            onDonationStateChange={this.onDonationStateChange}
+          />
+        </Col>
+        <Col lg={8} lgOffset={2} sm={10} smOffset={1} xs={12}>
+          <Spacer />
+          <b>Or donate with a credit card:</b>
+          <Spacer />
+          <StripeProvider stripe={stripe}>
+            <Elements>
+              <DonateFormChildViewForHOC
+                defaultTheme={defaultTheme}
+                donationAmount={donationAmount}
+                donationDuration={donationDuration}
+                getDonationButtonLabel={() =>
+                  `Confirm your donation of ${donationPlan}`
+                }
```

Comment:
```suggestion
                donationButtonLabel={`Confirm your donation of ${donationPlan}`}
```
Can we make this change and remove the other `getDonationButtonLabel` function?  I think `donationPlan` covers everything now.

Reply:
Thanks for bringing that into my attention. I would rather keep it around since we will be doing some A/B testing in near future: however, I removed the  donationPlan and replaced it with getDonation label to make it more concise.